WrathCombo/Combos/PvE/GNB/GNB_Config.cs

```diff
@@ -204,11 +229,11 @@ internal static void Draw(CustomComboPreset preset)
 
                     UserConfig.DrawHorizontalRadioButton(GNB_ST_Camouflage_SubOption,
                         "All Enemies",
-                        "Uses Camo regardless of targeted enemy type.", 1);
+                        "Uses Camouflage regardless of targeted enemy type.", 0);
```

Comment:
Better to use the localization method I employed in the WAR configs for naming actions rather than hard coding the string value. This applies to all things in the config that can come directly from the game.